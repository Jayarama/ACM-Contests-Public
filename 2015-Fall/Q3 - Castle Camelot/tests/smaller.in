rotate left
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
