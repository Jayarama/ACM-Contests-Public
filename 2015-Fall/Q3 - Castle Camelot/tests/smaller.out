zzzzzzzzzzzzzzzzzzzzzzzzzz
yyyyyyyyyyyyyyyyyyyyyyyyyy
xxxxxxxxxxxxxxxxxxxxxxxxxx
wwwwwwwwwwwwwwwwwwwwwwwwww
vvvvvvvvvvvvvvvvvvvvvvvvvv
uuuuuuuuuuuuuuuuuuuuuuuuuu
tttttttttttttttttttttttttt
ssssssssssssssssssssssssss
rrrrrrrrrrrrrrrrrrrrrrrrrr
qqqqqqqqqqqqqqqqqqqqqqqqqq
pppppppppppppppppppppppppp
oooooooooooooooooooooooooo
nnnnnnnnnnnnnnnnnnnnnnnnnn
mmmmmmmmmmmmmmmmmmmmmmmmmm
llllllllllllllllllllllllll
kkkkkkkkkkkkkkkkkkkkkkkkkk
jjjjjjjjjjjjjjjjjjjjjjjjjj
iiiiiiiiiiiiiiiiiiiiiiiiii
hhhhhhhhhhhhhhhhhhhhhhhhhh
gggggggggggggggggggggggggg
ffffffffffffffffffffffffff
eeeeeeeeeeeeeeeeeeeeeeeeee
dddddddddddddddddddddddddd
cccccccccccccccccccccccccc
bbbbbbbbbbbbbbbbbbbbbbbbbb
aaaaaaaaaaaaaaaaaaaaaaaaaa
